src/components/pages/CitySearch/Index.js

```diff
@@ -1,31 +1,50 @@
 import React from 'react';
 
 import logic from './logic.js';
+import dummyData from '../../../utils/dummyData';
 // import map from 'map';
 // import
 // import Header from '../../common/header';
 
 logic();
```

Comment:
What is this func doing? It's not very semantic. Might add a comment as to what's going on here for clarity.

Reply:
This is just a file I was using to simulate getting data from the backend. I'll remove it